include/GafferScene/Private/IECoreScenePreview/CapturingRenderer.h

```diff
@@ -84,6 +84,9 @@ class IECORESCENE_API CapturingRenderer : public Renderer
 
 				CapturedAttributes( const IECore::ConstCompoundObjectPtr &attributes );
 
+				int uneditableAttributeValue() const;
```

Comment:
What does it mean that uneditable is an int, rather than bool?

Reply:
Just that the attribute is of type `int`. The `unrenderable` attribute is an attribute to determine whether or not we pretend the object couldn't be rendered, so only has `True/False` values. The `uneditable` attribute emulates things like Arnold's `subdiv_iterations` attribute, where it can take on many values, but changing the value requires that the geometry is re-sent.